.travis.yml

```diff
@@ -7,33 +7,41 @@ cache:
 
 env:
   global:
-    - RUN_PHPUNIT=1
     - RUN_PHPCS=0
 
 matrix:
   include:
-    - php: 7.0
-      env: RUN_PHPUNIT=0
-    - php: 7.0
-      env: RUN_PHPUNIT=0 COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
```

Comment:
`--prefer-lowest` isn't necessary anymore due to the explicit php/laravel version matrix

Reply:
`--prefer-lowest` was actually re-introduced into the whole matrix!